boilerplate.js

```diff
@@ -155,24 +155,22 @@ async function install (context) {
   await system.spawn('react-native link', { stdio: 'ignore' })
   spinner.stop()
 
-  /* TODO: fix splash screen issues
   async function patchSplashScreen () {
     spinner.text = `▸ setting up splash screen`
     spinner.start()
     spinner.text = `▸ setting up splash screen: installing package`
-    await system.run(`yarn add react-native-splash-screen`, { stdio: 'ignore' })
+    await system.run(`yarn add react-native-splash-screen@3.0.6`, { stdio: 'ignore' })
```

Comment:
Did you know, we already have a way of doing this step?

If you add it in `package.json.ejs`, you can delete this line.

Reply:
I've tried but it wasn't installed somehow. Let me double check when I wake up. I'm sleepin'